solver/src/solver/http_solver.rs

```diff
@@ -484,23 +283,6 @@ impl HttpSolver {
             .with_context(|| format!("failed to decode response json, {}", context()))
     }
 
-    fn order_cost(&self, gas_price: f64) -> U256 {
```

Comment:
Moved :point_down: to member functions of `GasModel` to make it easier to unit test (specifically, its really easy to create a `GasModel` instance to test mapping of `LimitOrder`s and `Liquidity` to HTTP solver specific representations without needing a full `HttpSolver` instance).